### DISCUSSION TITLE
动态无法点开

### DISCUSSION THREAD

**Discussion by ChenJW431** [the1812/Bilibili-Evolved#4591] [提问 / Q&A] [answered]
动态无法点开，“全文”这个按钮失效，别人转发的动态也点不了。

**Comment by ChenJW431** [comment, marked as answer]
自问自答，用脚本https://greasyfork.org/zh-CN/scripts/472114-b%E7%AB%99%E8%87%AA%E5%8A%A8%E5%9B%9E%E5%88%B0%E6%97%A7%E7%89%88就可以莫名解决。

**Comment by GalaxySnail** [comment]
相关 issue：#4293
